scripts/template_upload_vsphere.py

```diff
@@ -78,7 +78,7 @@ def upload_ova(hostname, username, password, name, datastore,
                cluster, datacenter, url, provider, proxy,
                ovf_tool_client, default_user, default_pass):
 
-    cmd_args = ['ovftool --noSSLVerify']
+    cmd_args = list('ovftool --noSSLVerify')
```

Comment:
Initially I thought it was a PEP standard but I went back and checked, its not. Just a suggestion pycharm gives.  I see both styles throughout the codebase. Personally I guess I kinda like this style more.